data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,59 @@ mission "Remnant: Scent of Rain 1"
 			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
 			label scanningpayment
 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
+
+
+
+mission "Remnant: Will Not Someone Please Think Of The Children"
+	landing
+	name "Sympathy For The Korath"
+	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
+	minor
+	source
+		government "Remnant"
+		not attributes "station"
+	waypoint "Antevorta"
+	to offer
+		random < 30
+		"remnant: broken jump drive count" >= 10
+		or
+			and
+				has "Remnant: Cognizance 25: done"
+				not "flagship planet: Ssil Vida"
+			has "event: remnant: cognizance calm"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a sensitive request for you."`
+			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board?" his signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
+			choice
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have no intentions of risking my neck for some Korath."`
+					goto rathernot
+			`	"I am grateful for your understanding and willingness to help here. Together we can ensure everyone gets home safely."`
+				accept
+			label rathernot
+			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
+				decline
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Antevorta"
```

Comment:
The conversation says `Parca`, not `Antevorta`.
Also, how is this ship in Antevorta? That's not a trivial journey from the Korath Exile systems, especially not without a keystone of some sort (which the Exiles are either supposed to not have or not know how to use).
Is the idea that this ship had a key stone or similar but lost it?

Reply:
It was originally in Parca, but moved it to  Antevorta in response to this comment

https://github.com/endless-sky/endless-sky/pull/7698#issuecomment-1326566610

Fixing the text not being updated in all places though

Reply:
Yeah no exile keystones but mb it was the drive malfunctionning?

Reply:
Yeah will update the variant to stranded to make it clearer as to what happened.

Reply:
The drive malfunctioning allowing passage through the wormhole is certainly something.

Reply:
More like it teleported it far away and then broke

Reply:
But could be best to be in Parca still

Reply:
Honestly, it would probably make most sense for it to be trapped in the Postverta cluster.

Reply:
IE When you toggle the clusters state via the job "[De]Activate Ssil Vida Power"

Reply:
NVM just found the events, it looks like it updates the systems and not replaces them. Can change to that cluster then. Would make more sense.

https://github.com/endless-sky/endless-sky/blob/master/data/remnant/remnant%20events.txt#L344-L1132

Reply:
Maybe it should be stranded near Stercutus?

Reply:
Changed to Postverta cluster and added Ssil Vida as a spawn point for the side mission.